frame/bags-list/src/weights.rs

```diff
@@ -18,11 +18,11 @@
 //! Autogenerated weights for pallet_bags_list
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-01-05, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+//! DATE: 2022-01-19, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
 
 // Executed Command:
-// target/release/substrate
+// ./target/optimized/substrate
```

Comment:
`./target/production/substrate`?

Reply:
Yes will be replaced on the next run when someone updates the weights with the *bench-bot*.  
I locally named it `optimized` on the first run.  
Anyway, thanks for actually looking at the diff :smile: